syntaxes/flix.tmLanguage.json

```diff
@@ -100,13 +101,17 @@
                     "name": "keyword.control.match.flix",
                     "match": "\\b(case|match)\\b"
                 },
+                {
+                    "name": "keyword.control.try.flix",
+                    "match": "\\b(try|catch)\\b"
+                },
                 {
                     "name": "keyword.control.spawn.flix",
                     "match": "\\b(spawn)\\b"
                 },
                 {
-                    "name": "keyword.control.select.flix",
-                    "match": "\\b(select)\\b"
+                    "name": "keyword.control.chan.flix",
+                    "match": "\\b(chan)\\b"
```

Comment:
Actually `chan` is not a control keyword.

The issue with `select` is that it is used both as a control keyword:

```
def main(_args: Array[String]): Int & Impure = {
    let c1 = chan String 1;
    let c2 = chan String 1;
    let c3 = chan String 1;
    spawn mooo(c1, 0);
    spawn meow(c2, 3);
    spawn hiss(c3, 7);
    select {
        case m <- c1 => m |> println
        case m <- c2 => m |> println
        case m <- c3 => m |> println
    };
    0 // exit code
}
```


and as a non-control keyword:

```
/// Composes the facts and rules, and computes the lfp.
def main(_args: Array[String]): Int32 & Impure =
    query getFacts(), getRules() 
        select (x, y) from AncestorOf(x, y) |> println;
    0 // exit code
```


Anyway, TLDR: There should not be a `chan` keyword.